<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OVM_Dialog</class>
 <widget class="QWidget" name="OVM_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multi grid method (orbit flattening)</string>
  </property>
  <layout class="QGridLayout" columnstretch="1,1">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" stretch="0,1,0,0,0,0,0,0">
     <item>
      <widget class="QLabel">
       <property name="text">
        <string>Configuration:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_config">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_edit_conf">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel">
       <property name="text">
        <string>Mode:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radio_mode_x">
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radio_mode_y">
       <property name="text">
        <string>y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radio_mode_xy">
       <property name="text">
        <string>x/y</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox">
     <property name="title">
      <string>Monitor readouts:</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="TableView" name="mon_tab"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox">
     <property name="title">
      <string>Target values:</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="TableView" name="con_tab"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox">
     <property name="title">
      <string>Calculated steerer corrections</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="TableView" name="var_tab">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QPushButton" name="update_fit_button">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="execute_corrections">
          <property name="text">
           <string>Execute</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>madgui/widget/tableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
